modules/meryl/count/main.nf

```diff
@@ -0,0 +1,37 @@
+process MERYL_COUNT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::meryl=1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/meryl:1.3--h87f3376_1':
+        'quay.io/biocontainers/meryl:1.3--h87f3376_1' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*.meryldb"), emit: meryl_db
+    path "versions.yml"               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    for READ in $reads; do
```

Comment:
Is this to handle paired-end reads? Or something that should be parallelised via NF instead of using a `for` loop?

Reply:
Yes. Will add test for paired end data.